test/sequential/test-cluster-inspect-brk.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 const common = require('../common');
+if (!common.hasCrypto) {
```

Comment:
IMHO not having SSL is not a reason to skip this test, that's a regression.
What @targos suggested makes more sense, and should be reflected in title and commit message, i.e. `test:  skip cluster-inspect-brk if inspector is disabled`